src/Goblin.cpp

```diff
@@ -0,0 +1,25 @@
+#include "Goblin.hpp"
+#include "Rectangle.hpp"
+#include <raylib.h>
+
+const int Goblin::placeholderRadius = 15;
+
+const int Goblin::health { 10 };
+const int Goblin::damage { 1 };
+const float Goblin::damageCooldown { 1 };
+const float Goblin::speed { 80 };
+const raylib::Rectangle Goblin::collider { -placeholderRadius, -placeholderRadius, placeholderRadius * 2, placeholderRadius * 2 };
```

Comment:
the collider should have corner (0,0)

Reply:
otherwise itll start above and to the right of the enemy instead of on the enemy

Reply:
Nah, the collider should be like this since it's a circle. The `getCollider` has a little bug. I've fixed it.

Add  
```cpp
DrawRectangleLinesEx(getCollider(), 1.0f, RED);
```
in line 22 to test it